### PR TITLE
Add migration for Tex infoblox attributes

### PR DESCRIPTION
**Pull request by rhafer**
While we don't support the Infoblox integration yet. We need to accept the
attributes here to allow upgrading from Tex.

### CODE REVIEW

File: chef/data_bags/crowbar/template-neutron.json [hunk 1]
```diff
@@ -67,7 +106,7 @@
     "neutron": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 44,
+      "schema-revision": 46,
```
Comment (dirkmueller) [inline chef/data_bags/crowbar/template-neutron.json]: 45 perhaps ? or which other patch needs merging first?

Comment (rhafer) [inline chef/data_bags/crowbar/template-neutron.json]: @vuntz took 45 already in https://github.com/crowbar/crowbar-openstack/pull/216
So it rather depends on which one we want to merge first.

**Comment by rhafer** [comment]
rebased

File: chef/data_bags/crowbar/migrate/neutron/046_infoblox.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    a["use_infoblox"] = ta["use_infoblox"]
+  end
+  unless a.key? "infoblox"
+    a["infoblox"] = ta["infoblox"]
+  end
+  return a, d
+end
+
+def downgrade(ta, td, a, d)
+  a.delete("use_infoblox")
```
Comment (dirkmueller) [inline chef/data_bags/crowbar/migrate/neutron/046_infoblox.rb]: unless ta.key? "use_infoblox"
...

Comment (rhafer) [inline chef/data_bags/crowbar/migrate/neutron/046_infoblox.rb]: done

**Comment by vuntz** [comment]
+1

**Comment by dirkmueller** [comment]
+1

**Pull request merged**